Doc/library/traceback.rst

```diff
@@ -274,6 +292,13 @@ capture data for later printing in a lightweight fashion.
 
       For syntax errors - the offset into the text where the error occurred.
 
+   .. attribute:: end_offset
+
+      For syntax errors - the end offset into the text where the error occurred.
+      Can be ``None`` if not present.
+
+      .. versionadded:: 3.10
```

Comment:
It might be worth putting the 3.10 changes in a separate PR that can be backported all the way back to 3.10.

Reply:
Moved 3.10 changes to #105046